pluma/dialogs/pluma-preferences-dialog.c

```diff
@@ -199,6 +212,88 @@ auto_indent_checkbutton_toggled (GtkToggleButton        *button,
 	pluma_prefs_manager_set_auto_indent (gtk_toggle_button_get_active (button));
 }
 
+static void
+draw_spaces_checkbutton_toggled (GtkToggleButton        *button,
+                                 PlumaPreferencesDialog *dlg)
+{
+	int v;
```

Comment:
/nitpick
Instead of `int v;` this can be `DrawSpacesSettings v;` (and maybe change `v` to something more descriptive like `draw` or `setting` or whatever.

Reply:
Submitted a commit (b21161b2387d0859061aa8d4ada3daa5b179d74a) to address this.